lib/stripe/subscriptions/coupon.ex

```diff
@@ -28,7 +28,7 @@ defmodule Stripe.Coupon do
           max_redemptions: pos_integer | nil,
           metadata: Stripe.Types.metadata(),
           name: String.t() | nil,
-          percent_off: pos_integer | nil,
+          percent_off: number | nil,
```

Comment:
👍

Reply:
Would you mind changing `tax_percent` in invoice as well?

Reply:
No problem!